charon/config.py

```diff
@@ -39,6 +40,15 @@ def __init__(self, data: Dict):
         self.__ignore_signature_suffix: Dict = data.get("ignore_signature_suffix", None)
         self.__signature_command: str = data.get("detach_signature_command", None)
         self.__aws_cf_enable: bool = data.get("aws_cf_enable", False)
+        self.__amqp_queue: str = data.get("amqp_queue", None)
+        self.__sign_result_loc: str = data.get("sign_result_loc", None)
+        self.__quay_radas_auth_enabled: bool = data.get("quay_radas_auth_enabled", False)
+        self.__quay_radas_registry: str = data.get("quay_radas_registry", None)
+        self.__quay_radas_username: str = data.get("quay_radas_username", None)
+        self.__quay_radas_password: str = data.get("quay_radas_password", None)
+        self.__radas_sign_enabled: bool = data.get("radas_sign_enabled", False)
+        self.__radas_sign_timeout_count: int = data.get("radas_sign_timeout_count", 10)
+        self.__radas_sign_wait_interval_sec: int = data.get("radas_sign_wait_interval_sec", 60)
```

Comment:
What's the difference between the interval and timeout here?

Reply:
@ligangty timeout_count means the number of retries, interval means the time for per sleep, maybe rename them as 'radas_sign_timeout_retry_count' and 'radas_sign_timeout_retry_interval' would be more explicit.

Reply:
Sounds good.